towhee/functional/mixins/serve.py

```diff
@@ -21,8 +21,10 @@
 
 
 class _APIWrapper:
-    """
-    API Wrapper
+    """API Wrapper
```

Comment:
Can we hide the private classes and functions from the document? A function/class is considered to be private and should only be used in the same file it is defined if its name is started with "_".

Reply:
i think we can